content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -21,15 +19,16 @@ metrics that Mixer currently generates:
 
 To generate service-level metrics directly in the Envoy proxies, follow these steps:
 
-1.  To prevent duplicate telemetry generation, disable calls to `istio-telemetry` in the mesh:
-
+Option1:
     {{< text bash >}}
-    $ istioctl manifest apply --set values.mixer.telemetry.enabled=false,values.mixer.policy.enabled=false
+    $ istioctl manifest apply --set values.telemetry.enabled=true,values.telemetry.v2.enabled=true
     {{< /text >}}
 
-    {{< tip >}}
-    Alternatively, you can comment out `mixerCheckServer` and `mixerReportServer` in your [mesh configuration](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig).
-    {{< /tip >}}
+Option2:
+
+you can follow these steps manually:
+
+1. To disable mixer http reporting, set `disableMixerHttpReports` to true in your [mesh configuration](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig).
```

Comment:
do we need a bit more here (as `disableMixerHttpReports` is hidden)?

Reply:
We should mention that it is a peer property to `mixerCheckServer` since there will not be anything in the istio/istio config map.

Reply:
added more details, also created PR to add disableMixerHttpReports to istio/isto as well to avoid confusion